src/components/WebCLI/Output.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { screen } from "@testing-library/react";
+
+import { renderComponent } from "testing/utils";
+
+import Output, { ansiColors } from "./Output";
+
+describe("Output", () => {
+  it("should display content and not display help message", () => {
+    renderComponent(
+      <Output
+        content="Output"
+        helpMessage="Help message"
+        showHelp={false}
+        setShouldShowHelp={jest.fn()}
+      />,
+    );
+    expect(screen.getByText("Output")).toBeInTheDocument();
+    expect(screen.queryByText("Help message")).not.toBeInTheDocument();
+  });
+
+  it("should not display content and display help message", () => {
+    renderComponent(
+      <Output
+        content="Output"
+        helpMessage="Help message"
+        showHelp={true}
+        setShouldShowHelp={jest.fn()}
+      />,
+    );
+    expect(screen.queryByRole("Output")).not.toBeInTheDocument();
+    expect(screen.getByText("Help message")).toBeInTheDocument();
+  });
+
+  it("should display the content with correct color", () => {
```

Comment:
I think it would be good to add a test for strings starting with a colour.

Reply:
I've changed the code to use `ansi-to-html` library instead of the previous custom code. Should we still add this test, considering that it is handled internally by the library? I've updated the previous test to check for bold and colored text.

Reply:
No need to add that test now.